deployments/common/image/common-env/common.pip

```diff
@@ -7,6 +7,7 @@ pyds9
 pipdeptree
 pip-tools
 pydeps
-graphviz
```

Comment:
Is this not needed?

Reply:
That's the coming change I mentioned.  It's needed by tools/image-graph-env to do dependency plots.   But there are issues installing it with conda vs. pip since it is named differently in each.  So my planned approach is to install it on the fly whenever someone wants to do a plot.   Still building and testing that.

Reply:
Got it, thanks.